pkg/suggestion/v1alpha3/hyperopt_service.py

```diff
@@ -2,15 +2,18 @@
 
 from pkg.apis.manager.v1alpha3.python import api_pb2
 from pkg.apis.manager.v1alpha3.python import api_pb2_grpc
+from pkg.apis.manager.health.python import health_pb2
+
 from pkg.suggestion.v1alpha3.internal.search_space import HyperParameter, HyperParameterSearchSpace
 from pkg.suggestion.v1alpha3.internal.trial import Trial, Assignment
 from pkg.suggestion.v1alpha3.hyperopt.base_hyperopt_service import BaseHyperoptService
+from pkg.suggestion.v1alpha3.base_health_service import HealthServicer
 
 logger = logging.getLogger("HyperoptRandomService")
 
 
 class HyperoptService(
-        api_pb2_grpc.SuggestionServicer):
+        api_pb2_grpc.SuggestionServicer, HealthServicer):
```

Comment:
With this change, it is mandatory for all suggestion service to use HealthServices. Right?

Reply:
Yes, we need to add health servicer to all algorithms

Reply:
Sounds good. We should add them in the suggestion doc separately. Also, Dockerfile instructions to have graph-health-check binary also has to be documented.